src/main/java/com/mycompany/myapp/web/rest/pool/PoolResource.java

```diff
@@ -48,4 +52,32 @@ public PoolResource(PoolService poolService) {
             return ResponseEntity.status(HttpStatus.NOT_ACCEPTABLE).body(null);
         }
     }
+
+    @PostMapping("/pool/save")
```

Comment:
Well even after clicking all of the dots, I get the error: `java.lang.IllegalStateException: io.undertow.server.RequestTooBigException: UT000020: Connection terminated as request was larger than 262144`